Modix.Bot/Behaviors/ModerationLoggingBehavior.cs

```diff
@@ -122,7 +117,7 @@ internal protected IModerationService ModerationService
                 { (ModerationActionType.InfractionDeleted,   InfractionType.Mute),    "`[{0}]` **{1}** deleted a mute (`{2}`) for **{3}** (`{4}`)" },
                 { (ModerationActionType.InfractionDeleted,   InfractionType.Ban),     "`[{0}]` **{1}** deleted a ban (`{2}`) for **{3}** (`{4}`)" },
                 { (ModerationActionType.MessageDeleted,      null),                   "`[{0}]` **{1}** deleted the following message (`{6}`) from **{7}** (`{8}`) in **#{9}** ```\n{12}``` for reason ```\n{11}```" },
-                { (ModerationActionType.MessageBatchDeleted, null),                  $"`[{{0}}]` **{{1}}** deleted **{{13}}** messages in **#{{9}}** (<{_batchDeletedUrl}>)" },
+                { (ModerationActionType.MessageBatchDeleted, null),                   "`[{0}]` **{1}** deleted **{13}** messages in **#{9}** (<{14}>)" },
```

Comment:
You know, this is a lot clearer and avoids the strange inconsistent string interpolation. I hadn't seen the `var message` block above when I made this change, but this is a lot cleaner imo.